subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,6 +1,9 @@
 The Gradle team is excited to announce Gradle 5.2.
 
-This release features [1](), [2](), ... [n](), and more.
+This release features the [Java Platform plugin](userguide/java_platform_plugin.html) which allows users to declare and publish platforms including as BOMs, [official support for additional Windows native toolchains](userguide/native_software.html#native-binaries:tool-chain-support), and more.
```

Comment:
I just added another blurb about the new C++ plugins, I think we should promote those here over the mingw/cygwin stuff.  I think those used to work before, but we just had no coverage for them really.

I would say something like...
> This release features the [Java Platform plugin](linky), which allows you to declare a platform.  Like Maven BOMs, platforms can be used to define a set of versions for dependencies that are known to work together. These versions can be published and consumed elsewhere as dependency recommendations.

@ljacomet does this sound about right? ☝️ Is the Java Platform plugin chapter referenced anywhere in the navigation sidebar?

and continue with...
> This release also has a new set of [C++ plugins](userguide/cpp_plugins.html) that replace the software model C++ plugins. You can go for a spin by running `gradle init` and following the instructions to generate a C++ project. We recommend that C++ projects use the `cpp-application` or `cpp-library` plugin instead of the `cpp` plugin when starting new projects.

Reply:
@big-guy That's a really good suggestion for platforms. 👍 from me.
The new plugin is not referenced in the navigation sidebar at all.
I am going top open a PR that will add a section similar to [this one](https://docs.gradle.org/nightly/userguide/building_java_projects.html#sec:building_java_libraries)

Reply:
In that case, I believe that dropping
> including as BOMs
from the sentence reads better. After all, once Gradle Metadata is around, this feature is more than BOMs.

Reply:
👍 I already changed it.

Reply:
PR for having the reference to platforms in build java projects: #8329